src/index.js

```diff
@@ -47,7 +47,22 @@ export function update({
 //TBR: remove when modules can specify their own optional system dependencies
 /** @type {Record<string, any> } */
 const additionalModuleDependencies = {
-    nro: { appserver: [], tools: ['osm'] }
+    network_revenue_optimizer: { appserver: [], tools: ['osm'] }
+};
+
+//schema version names for modules are not consistently named, so we need a mapping
+/** @type {Record<string, any> } */
+const moduleSchemaVersionNames = {
+    capture: 'capture_schema',
+    workflow_manager: 'mywmywwfm_schema',
+    survey: 'mywis_schema',
+    gas: 'mywgas_schema',
+    electric: 'iqg_electric_schema',
+    comms: 'myw_comms_schema',
+    comsof: 'iqg_comsof_schema',
+    comms_cloud: 'iqg_comms_cloud_schema',
+    network_revenue_optimizer: 'mywnro_schema',
+    pia_interface: 'myw_pia_schema'
```

Comment:
Would it be worth making separate typedefs for these two objects?

If not, we could just be a bit more specific with the typing:

```js
/** @type {Record<string, string> } */
const moduleSchemaVersionNames = {
```

Reply:
for the first one, it would be worth it, yes. For the second one, just what you suggested. Thanks